//= error code 1

fun main() {
    std::exit(1i32);
}